day-2/src/App.js

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react';
+import logo from './logo.svg';
+import './App.css';
+
+var quotes = ["Just hire a lawyer", "Empower your team", "Raise your prices", "Ignore most of the advice you're given", "Stop trying to hire a unicorn"];
+
+
+
+class App extends Component {
+
+  constructor (props) {
+    super (props);
+    this.state = {
+      attribute: quotes,
+      currentIndex: 0,
+    };
+  }
+
+  rotateQuotes(quotes) {
+      if ( this.state.currentIndex === 4) {
+        this.setState( {
+          attributes: quotes, 
```

Comment:
`attributes` is quite vague... also is this necessary if quotes is a class variable?